example/devstone/parallel_main_devstone.cpp

```diff
@@ -35,7 +35,7 @@ int main(int argc, char *argv[]) {
 
 	// Then, we generate the corresponding DEVStone model and inject the original
 	auto coupled = DEVStoneCoupled::newDEVStoneCoupled(type, width, depth, intDelay, extDelay);
-	coupled->flatten();
+	// coupled->flatten();
```

Comment:
Do we have to flatten the model when using parallel coordinator or does the flattening trigger automatically?